src/js/components/ToggleButtonGroup/stories/multiple.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, ToggleButtonGroup } from 'grommet';
+
+export const Multiple = () => (
+  <Box pad="large">
+    <ToggleButtonGroup options={['Red', 'Green', 'Blue']} multiple />
```

Comment:
Let's make these examples more representative of real world use cases. This one seems like it lends itself to a text formatting use case e.g. "bold, italics, underline."